ui/admin/package.json

```diff
@@ -44,6 +44,7 @@
     "@babel/core": "^7.26.10",
     "@babel/eslint-parser": "^7.25.1",
     "@babel/plugin-proposal-decorators": "^7.24.7",
+    "@ember-data/store": "^5.4.1",
```

Comment:
after upgrading ember in ui/admin i was getting this error:
![image](https://github.com/user-attachments/assets/302a29df-007d-4641-b41b-4328e0f0d468)
so i had to import @ember-data/store to resolve it